pandas/io/sql.py

```diff
@@ -1404,6 +1404,15 @@ class SQLiteTable(SQLTable):
     Instead of a table variable just use the Create Table statement.
     """
 
+    def __init__(self, *args, **kwargs):
+        # GH 8341
+        # register an adapter callable for datetime.time object
+        import sqlite3
```

Comment:
Done... I think, see  the test below

@jorisvandenbossche is it what you had in mind?